<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameBrowser</class>
 <widget class="QMainWindow" name="FrameBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>:ScDrv[*]:</string>
  </property>
  <widget class="FrameFile" name="tabViews_"/>
  <widget class="QMenuBar" name="barMenu_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_">
     <property name="title">
      <string>Recen&amp;t..</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuExport_">
     <property name="title">
      <string>&amp;Export..</string>
     </property>
     <addaction name="actionExportScene_"/>
     <addaction name="actionExportCoords_"/>
    </widget>
    <addaction name="actionNew_"/>
    <addaction name="actionClone_"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_"/>
    <addaction name="actionReload_"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_"/>
    <addaction name="separator"/>
    <addaction name="actionSave_"/>
    <addaction name="actionSaveAs_"/>
    <addaction name="menuExport_"/>
    <addaction name="separator"/>
    <addaction name="actionClose_"/>
    <addaction name="actionExit_"/>
   </widget>
   <widget class="QMenu" name="menuHelp_">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAbout_"/>
    <addaction name="actionAboutQt_"/>
   </widget>
   <widget class="QMenu" name="menuEdit_">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo_"/>
    <addaction name="actionRedo_"/>
    <addaction name="separator"/>
    <addaction name="actionCut_"/>
    <addaction name="actionCopy_"/>
    <addaction name="actionPaste_"/>
    <addaction name="separator"/>
    <addaction name="actionClear_"/>
    <addaction name="actionClearAll_"/>
   </widget>
   <widget class="QMenu" name="menuView_">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuMolecule_">
     <property name="title">
      <string>&amp;Molecule</string>
     </property>
     <addaction name="actionMolFast_"/>
     <addaction name="separator"/>
     <addaction name="actionMolStick_"/>
     <addaction name="actionMolBalls_"/>
     <addaction name="actionMolSpace_"/>
     <addaction name="separator"/>
     <addaction name="actionMolBonds_"/>
     <addaction name="actionLabelAtoms_"/>
     <addaction name="actionLabelBonds_"/>
    </widget>
    <widget class="QMenu" name="menuProject_">
     <property name="title">
      <string>Pro&amp;jection</string>
     </property>
     <addaction name="actionProjPerspective_"/>
     <addaction name="actionProjOrthogonal_"/>
    </widget>
    <addaction name="actionToggleLayout_"/>
    <addaction name="actionFullScreen_"/>
    <addaction name="separator"/>
    <addaction name="menuProject_"/>
    <addaction name="separator"/>
    <addaction name="menuMolecule_"/>
    <addaction name="separator"/>
    <addaction name="actionPrintNow_"/>
    <addaction name="actionPrintSetup_"/>
    <addaction name="actionImageExport_"/>
   </widget>
   <widget class="QMenu" name="menuTools_">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSetFont_"/>
    <addaction name="actionSourceEdit_"/>
    <addaction name="actionSourceCast_"/>
    <addaction name="separator"/>
    <addaction name="actionBoxCenter_"/>
    <addaction name="separator"/>
    <addaction name="actionOptions_"/>
   </widget>
   <widget class="QMenu" name="menuWindows_">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFormat_">
    <property name="title">
     <string>&amp;Format</string>
    </property>
    <addaction name="actionPageSetup_"/>
   </widget>
   <addaction name="menuFile_"/>
   <addaction name="menuEdit_"/>
   <addaction name="menuView_"/>
   <addaction name="menuFormat_"/>
   <addaction name="menuTools_"/>
   <addaction name="menuWindows_"/>
   <addaction name="menuHelp_"/>
  </widget>
  <widget class="QStatusBar" name="barStatus_"/>
  <widget class="QToolBar" name="barTools_">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;New..</string>
   </property>
  </action>
  <action name="actionOpen_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open..</string>
   </property>
  </action>
  <action name="actionPrint_">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp_">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionSave_">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit_">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout_">
   <property name="text">
    <string>&amp;About..</string>
   </property>
  </action>
  <action name="actionAboutQt_">
   <property name="text">
    <string>About &amp;Qt..</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionUndo_">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo_">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCut_">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy_">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste_">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionClear_">
   <property name="text">
    <string>C&amp;lear</string>
   </property>
  </action>
  <action name="actionClearAll_">
   <property name="text">
    <string>Cle&amp;ar all</string>
   </property>
  </action>
  <action name="actionReload_">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionForget_">
   <property name="text">
    <string>F&amp;orget this</string>
   </property>
  </action>
  <action name="actionForgetAll_">
   <property name="text">
    <string>Forget &amp;all recent</string>
   </property>
  </action>
  <action name="actionFullScreen_">
   <property name="text">
    <string>&amp;Full screen</string>
   </property>
  </action>
  <action name="actionOptions_">
   <property name="text">
    <string>&amp;Options..</string>
   </property>
  </action>
  <action name="actionBoxCenter_">
   <property name="text">
    <string>Bo&amp;x center</string>
   </property>
  </action>
  <action name="actionSaveAs_">
   <property name="text">
    <string>S&amp;ave as..</string>
   </property>
  </action>
  <action name="actionToggleLayout_">
   <property name="text">
    <string>Togg&amp;le layout</string>
   </property>
  </action>
  <action name="actionMolFast_">
   <property name="text">
    <string>Render &amp;fast</string>
   </property>
  </action>
  <action name="actionMolStick_">
   <property name="text">
    <string>&amp;Sticks</string>
   </property>
  </action>
  <action name="actionMolSpace_">
   <property name="text">
    <string>S&amp;pace fill</string>
   </property>
  </action>
  <action name="actionMolBalls_">
   <property name="text">
    <string>&amp;Balls and sticks</string>
   </property>
  </action>
  <action name="actionMolBonds_">
   <property name="text">
    <string>Toggle bonds display</string>
   </property>
   <property name="statusTip">
    <string>Toggle the bond display in the molecule</string>
   </property>
  </action>
  <action name="actionLabelAtoms_">
   <property name="text">
    <string>Atom labels</string>
   </property>
   <property name="statusTip">
    <string>Toggle the atom label display in the molecule</string>
   </property>
  </action>
  <action name="actionLabelBonds_">
   <property name="text">
    <string>Bond labels</string>
   </property>
  </action>
  <action name="actionSourceEdit_">
   <property name="text">
    <string>&amp;Edit source..</string>
   </property>
  </action>
  <action name="actionSourceCast_">
   <property name="text">
    <string>&amp;Interpret..</string>
   </property>
  </action>
  <action name="actionPrintNow_">
   <property name="text">
    <string>&amp;Print..</string>
   </property>
  </action>
  <action name="actionPrintSetup_">
   <property name="text">
    <string>Print &amp;setup..</string>
   </property>
  </action>
  <action name="actionImageExport_">
   <property name="text">
    <string>E&amp;xport..</string>
   </property>
  </action>
  <action name="actionPageSetup_">
   <property name="text">
    <string>&amp;Page..</string>
   </property>
  </action>
  <action name="actionExportScene_">
   <property name="text">
    <string>&amp;Scene..</string>
   </property>
  </action>
  <action name="actionExportCoords_">
   <property name="text">
    <string>&amp;Coordnates..</string>
   </property>
  </action>
  <action name="actionProjPerspective_">
   <property name="text">
    <string>&amp;Perspective</string>
   </property>
  </action>
  <action name="actionProjOrthogonal_">
   <property name="text">
    <string>Orthogonal</string>
   </property>
  </action>
  <action name="actionSetFont_">
   <property name="text">
    <string>Set text &amp;font..</string>
   </property>
  </action>
  <action name="actionClone_">
   <property name="text">
    <string>C&amp;lone</string>
   </property>
   <property name="toolTip">
    <string>Clone current content into a new file</string>
   </property>
  </action>
  <action name="actionClose_">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close this window</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameFile</class>
   <extends>QTabWidget</extends>
   <header>FrameFile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
